docs/contracts/functions.rst

```diff
@@ -107,7 +108,8 @@ Returning Multiple Values
 -------------------------
 
 When a function has multiple return types, the statement ``return (v0, v1, ..., vn)`` can be used to return multiple values.
-The number of components must be the same as the number of return types.
+The number of components must be the same as the number of return variables
+and their types have to match, potentially after an implicit conversion.
```

Comment:
I think there's a section on implicit conversion, maybe link to it? Actually I checked, it has no link defined, but I can add to this PR if you think it's a useful idea

Reply:
yes, please do!

Reply:
Done, ended up using an existing link is close enough anyway